keno/keno_input.inp

```diff
@@ -0,0 +1,431 @@
+media 1190502 1 -1000 1001 1102 -1103
+media 1190599 1 -1001 1002 1100 -1103
+media 50002 1 1002 2003 -2004
+media 50002 1 -1001 1002 1103 -2003
+media 30001 1 1001 1103 -2003
+media 20000 1 1000 1100 -1103
+media 30001 1 1001 2002 -1100
+media 50002 1 -1001 1002 2002 -1100
+media 50001 1  1002 2000 -2002
+media 30000 1 -1002 1
+media 30000  1 -2000 1
+media 30000 1 2004
+boundary 1
+unit 2203
+com="copy pin Row:2 Column:2 Pin:3"
+Zcylinder 1000 0.5 15 0 
+Zcylinder 1001 1.5 15 0
+Zcylinder 1002 2 15 0 
+plane 1100 Zpl=1 Con=-3
+plane 1101 Zpl=1 Con=-6
+plane 1102 Zpl=1 Con=-9
+plane 1103 Zpl=1 Con=-12
+plane 2000 Zpl=1 Con=0
+plane 2002 Zpl=1 Con=-1
+plane 2003 Zpl=1 Con=-14
+plane 2004 Zpl=1 Con=-15
+cuboid 1 2.5 -2.5 2.5 -2.5 15  0
+media 1190600  1 -1000 1001 1100 -1101
+media 1190601 1 -1000 1001 1101 -1102
+media 1190602 1 -1000 1001 1102 -1103
+media 1190599 1 -1001 1002 1100 -1103
+media 50002 1 1002 2003 -2004
+media 50002 1 -1001 1002 1103 -2003
+media 30001 1 1001 1103 -2003
+media 20000 1 1000 1100 -1103
+media 30001 1 1001 2002 -1100
+media 50002 1 -1001 1002 2002 -1100
+media 50001 1  1002 2000 -2002
+media 30000 1 -1002 1
+media 30000  1 -2000 1
+media 30000 1 2004
+boundary 1
+unit 2204
+com="copy pin Row:2 Column:2 Pin:4"
+Zcylinder 1000 0.5 15 0 
+Zcylinder 1001 1.5 15 0
+Zcylinder 1002 2 15 0 
+plane 1100 Zpl=1 Con=-3
+plane 1101 Zpl=1 Con=-6
+plane 1102 Zpl=1 Con=-9
+plane 1103 Zpl=1 Con=-12
+plane 2000 Zpl=1 Con=0
+plane 2002 Zpl=1 Con=-1
+plane 2003 Zpl=1 Con=-14
+plane 2004 Zpl=1 Con=-15
+cuboid 1 2.5 -2.5 2.5 -2.5 15  0
+media 1190700  1 -1000 1001 1100 -1101
+media 1190701 1 -1000 1001 1101 -1102
+media 1190702 1 -1000 1001 1102 -1103
+media 1190799 1 -1001 1002 1100 -1103
+media 50002 1 1002 2003 -2004
+media 50002 1 -1001 1002 1103 -2003
+media 30001 1 1001 1103 -2003
+media 20000 1 1000 1100 -1103
+media 30001 1 1001 2002 -1100
+media 50002 1 -1001 1002 2002 -1100
+media 50001 1  1002 2000 -2002
+media 30000 1 -1002 1
+media 30000  1 -2000 1
+media 30000 1 2004
+boundary 1
+unit 13100
+com="Fuel pin lattice"
+cuboid 1 5 -5 5 -5 15 0
+array 1 1 place  1 1 1 -2.5 -2.5 0
+boundary 1
+unit 31
+com="fuel bundle Row:3 Column:1"
+plane 115 xpl=1 con=-5.0
+plane 116 xpl=1 con=5.0
+plane 117 ypl=1 con=-5.0
+plane 118 ypl=1 con=5.0
+plane 2000 zpl=1 con=0.0
+cuboid 1 5.0 -5.0 5.0 -5.0 40 -10
+hole 13100 origin x=0 y=0 z=0
+hole 20000  origin x=0 y=0 z=-6
+media 30000 1 1
+boundary 1
+unit 20000
+com="Bottom adapter"
+plane 201 xpl=1 con=-3
+plane 202 xpl=1 con=3
+plane 203 ypl=1 con=-3
+plane 204 ypl=1 con=3
+plane 205 zpl=1 con=-6
+plane 2000 zpl=1 con=0
+cuboid 1 3 -3 3 -3 6 0
+media 50001 1 -201 202 -203 204 -205 2000
+media 30000 1 201
+media 30000 1 -202 1 
+media 30000 1 203
+media 30000 1 -204 1
+media 30000 1 205  
+media 30000 1 -2000 1
+boundary 1
+unit 3101
+com="Transient rod"
+cylinder 4541 1.8 20 0
+plane 4542 zpl=1 con=0 
+plane 4543 zpl=1 con=20.0 
+media 40001 1 4541
+boundary 4541
+unit 12200
```

Comment:
This is a comment to try to make numbers consistent among input files:
From the proposed model geometry, a 3x2 grid can be imagined in the core, ignoring the control shroud.
Therefore, in MCNP input, 13100 was meant to represent the fuel pin lattice(or bundle) located in row 3, column 1, which is bottom left one,
while 12200 represents the one located in row 2, column 2, which is right middle one.

I guess you switched the two numbers, 31s and 22s.